P202.Training.WCF/Web.config

```diff
@@ -1,6 +1,20 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
-
+  <configSections>
+  <section
+         name="hibernate-configuration"
+         type="NHibernate.Cfg.ConfigurationSectionHandler, NHibernate"
+        />  
+  </configSections>
+  <!-- Add this element -->
+  <hibernate-configuration xmlns="urn:nhibernate-configuration-2.2">
+    <session-factory>
+      <property name="dialect">NHibernate.Dialect.MsSql2005Dialect</property>
+      <property name="connection.provider">NHibernate.Connection.DriverConnectionProvider</property>
+      <property name="connection.connection_string">Server=TENAVAROLAP\TENAVAROSQL;initial catalog=TrainingNH;Integrated Security=True</property>
```

Comment:
Make sure you update this connection string to the localdb version before merging